### PR TITLE
Use computed for store access simplify. Close: #4

### PR DESCRIPTION
**Pull request by liubich**

### CODE REVIEW

File: src/components/CurrentWeather.vue [hunk 1]
```diff
@@ -1,46 +1,39 @@
...
       </div>
       <div class="current-weather__right-column column">
         <div class="current-weather__wind-container">
           <div class="current-weather__wind-caption">вітер:</div>
           <div
-            v-if="this.$store.state.currentWeather.windSpeed"
+            v-if="currentWeather.windSpeed"
             :style="{transform:
-            'rotate('+ this.$store.state.currentWeather.windDirectionDeg+'deg)'}"
+            'rotate('+ currentWeather.windDirectionDeg+'deg)'}"
```
Comment (probil) [inline src/components/CurrentWeather.vue]: you can use ES6 inside templates:
```vue
<div
:style="{
  transform: `rotate(${currentWeather.windDirectionDeg}deg)`
}"
```
As alternative you can create computed property for that

File: src/components/CurrentWeather.vue [hunk 2]
```diff
@@ -50,8 +43,10 @@
 <script>
 export default {
   name: 'CurrentWeather',
-  props: {
-    currentWeather: Object,
+  computed: {
```
Comment (probil) [inline src/components/CurrentWeather.vue]: I prefer to use props here.
It's presentational component

File: src/components/CurrentWeather.vue [hunk 3]
```diff
@@ -50,8 +43,10 @@
 <script>
 export default {
```
Comment (probil) [inline src/components/CurrentWeather.vue]: Consider using functional component

File: babel.config.js [hunk 4]
```diff
@@ -1,5 +1,3 @@
 module.exports = {
-  presets: [
-    '@vue/app',
-  ],
+  presets: [['@vue/app', { useBuiltIns: 'entry' }]],
```
Comment (probil) [inline babel.config.js]: Why do you need `{ useBuiltIns: 'entry' }`?

Comment (liubich) [reply babel.config.js]: I just copied this from [here](https://github.com/vuejs/vue-cli/issues/3678) as a solution of my build problem and it`s works.

Comment (probil) [reply babel.config.js]: Ok but I think this fix should be in separate branch ;)

File: src/components/CurrentWeather.vue (hunk 3 above)
Comment (liubich) [reply src/components/CurrentWeather.vue]: @probil, what is the reason of using functional components?

File: babel.config.js (hunk 4 above)
Comment (liubich) [reply babel.config.js]: I removed this change

File: src/components/CurrentWeather.vue (hunk 3 above)
Comment (probil) [reply src/components/CurrentWeather.vue]: The same reason as function components in react. They are cheaper to render. They are lightweight. They are just functions (converted to function under the hood).

**Comment by probil** [comment]
rebase to fix build error

**Pull request merged**